src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -1154,10 +1155,28 @@ public function edit(TwillModelContract|int $id, $submoduleId = null)
             }
         }
 
-        return View::make($view, $this->form($id))->with(
-            'renderFields',
-            $controllerForm
-        );
+        $viewWithData = [];
```

Comment:
Can we move this out of the edit method maybe? Just to avoid it from growing.

Perhaps a `buildFormBuilderData(TwillModelContract $item): array` method

Reply:
Yes, definitely

Reply:
Hey, I have merged https://github.com/area17/twill/pull/1971 so we also have to apply this there, so the abstraction is for sure needed :).